chrome/patches/curl-impersonate.patch

```diff
@@ -1212,6 +1229,48 @@ index 000000000..2c8a4d3f9
 +    .http2_no_server_push = true
 +  },
 +  {
++          .target = "chrome107",
```

Comment:
Should be: `"chrome109"`.

This entry is also indented more than other entries.